spec/models/allyship_spec.rb

```diff
@@ -49,4 +49,83 @@
 	  new_allies = build(:allyships_accepted, user_id: new_user1.id, ally_id: nil)
 	  expect(new_allies).to have(1).error_on(:ally_id)
 	end
+  context 'when destroying' do
```

Comment:
Looks like we're testing `remove_ally_notifications` which is good, but we should add tests for `remove_ally_viewers` :D

Reply:
I forgot to add this test, thank you for catching this!